src/components/app/TabBar.js

```diff
@@ -43,9 +49,23 @@ class TabBar extends React.PureComponent<Props> {
               })}
               key={tabSlug}
               data-name={tabSlug}
-              onMouseDown={this._mouseDownListener}
+              onClick={this._onClickListener}
             >
-              {tabsWithTitle[tabSlug]}
+              {/* adding a button for better keyboard navigation  
+            (allows tabbing between the different panels)
+            ARIA attributes for better accessibility and usability
+            (arrow navigation between the tabs) */}
+              <button
+                className="tapBarTabButton"
+                type="button"
+                id={tabSlug}
+                role="tab"
+                aria-selected={tabSlug === selectedTabSlug}
+                aria-controls={`${tabSlug}-tab`}
+                tab-index={tabSlug === selectedTabSlug ? null : -1}
```

Comment:
isn't it `tabIndex`? _But_ we don't want this now, because otherwise we won't be able to tab to the other tab titles :) So it's a good thing that this doesn't work!